assignment6.txt

```diff
@@ -0,0 +1,18 @@
+Mariposa Lee
+
+1. I'm not sure what the code is trying to do, but every time the for loop runs, the array elements would all be zero. Unless they were trying to aim for that, I don't necessarily see any other errors. It would be nice for there to be a '\n' at the end of "enter the size of an array". But as of now, when the code runs, it will ask the user for the length of an array, and apply 0 to each element up to the size the user inputted. 
```

Comment:
So actually, this code would cause a seg fault. What you described is what it would do if it didn't seg fault. The reason for the seg fault is that the size of the array should be known before the array declaration. To achieve this, you can just move the array declaration until AFTER the size input is scanned.